clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -138,6 +139,7 @@ public enum ClientType {
   private final boolean writeBatchingPushStatus;
   private final HeartbeatMonitoringService heartbeatMonitoringService;
   private AggVersionedBlobTransferStats aggVersionedBlobTransferStats;
+  private final Map<String, Store> cachedStores = new ConcurrentHashMap<>();
```

Comment:
Looks like we already have `storeRepository`, do we still need to maintain another cache here, `cachedStores`?